doc/source/conf.py

```diff
@@ -19,43 +19,39 @@
 # If extensions (or modules to document with autodoc) are in another directory,
 # add these directories to sys.path here. If the directory is relative to the
 # documentation root, use os.path.abspath to make it absolute, like shown here.
-sys.path.insert(0, os.path.abspath('../..'))
+sys.path.insert(0, os.path.abspath("../.."))
```

Comment:
Whoever did this deserves to die.

Reply:
I cant comment on cppsal. This pull request is for the csal interface.

Reply:
I have not touched any python code yet.  Although some bugs are found

Reply:
I suspect it must have been a misconfigured auto reformat tool.